src/cards/alarm-control-panel-card/alarm-control-panel-card.ts

```diff
@@ -165,9 +167,9 @@ export class AlarmControlPanelCard extends LitElement implements LovelaceCard {
 
         return html`
             <ha-card>
-                <div class="container">
+                <mushroom-card .layout=${layout} no-card-style="true">
```

Comment:
```suggestion
                <mushroom-card .layout=${layout} no-card-style>
```

setting the attribute is the same as true